app/main.py

```diff
@@ -1,6 +1,28 @@
-# write your imports here
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(
+        customers: list,
+        hall_number: int,
+
+        cleaner: str,
+        movie: str
+) -> None:
+
+    customer_instances = [
+        Customer(
+            cust["name"],
+            cust["food"]
+        ) for cust in customers
```

Comment:
you had shortened few letters, it doesn't make a big difference, but full variable names would be more readable
```suggestion
        Customer(
            customer["name"],
            customer["food"]
        ) for customer in customers
```